Docker/download_medical_models.py

```
@@ -0,0 +1,203 @@
+    model_dir.mkdir(parents=True, exist_ok=True)
+    destination = model_dir / file_name
+
+    if destination.exists():
+        return destination
+
+    file_url = f"{HF_WEB_BASE}/{model_id}/resolve/main/{quote(file_name)}?download=true"
+    headers = {"User-Agent": "llm-server-medical-model-fetcher/1.0"}
+    if token:
+        headers["Authorization"] = f"Bearer {token}"
+
+    request = Request(file_url, headers=headers)
```

Ruff (0.15.7)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)